src/keys/list.ts

```diff
@@ -71,6 +82,9 @@ const listKeymap: { [key: string]: EditorAction } = {
   'Mod-Alt-k': skipCommandTracking(wrapInList(schema.nodes.bullet_list)),
   'Shift-Tab': skipCommandTracking(liftToOuterList(schema.nodes.list_item)), // outdent, same as Mod-[
   Tab: skipCommandTracking(sinkListItem(schema.nodes.list_item)), // indent, same as Mod-]
+  Backspace: skipCommandTracking(
+    chainCommands(undoInputRule, joinBackward, deleteSelection)
+  ),
```

Comment:
with that we are going to ignore Backspace for [section title](https://github.com/Atypon-OpenSource/manuscripts-body-editor/blob/5059c73334379c8d727bba88c2931a65f3dffea2/src/keys/title.ts#L233) and [atom node](https://github.com/Atypon-OpenSource/manuscripts-body-editor/blob/5059c73334379c8d727bba88c2931a65f3dffea2/src/keys/misc.ts#L45). so we could check the selection if it's related to list item then do `joinBackward` or return false to check other Backspace commands. we can do it like this:

`Backspace: listItemBackward`

```
const listItemBackward = (
  state: EditorState,
  dispatch?: (tr: Transaction) => void
) => {
  const { selection } = state
  const selectionParentNode = state.selection.$from.node(
    selection.$from.depth - 1
  )
  if (selectionParentNode.type === schema.nodes.list_item) {
    return skipCommandTracking((state, dispatch) => joinBackward(state, dispatch))(
      state,
      dispatch
    )
  }

  return false
}
```